homeassistant/components/developer_credentials/manifest.json

```diff
@@ -0,0 +1,9 @@
+{
+  "domain": "developer_credentials",
+  "name": "Developer Credentials",
```

Comment:
I would opt to remove/avoid the term "developer". It might sound scary to less technical end-users. In OAuth specs, this is called "Client Credentials" and more often referred to as "Application Credentials" too (as most platforms, like Spotify, Twitter, and more, you create "Applications" on their portals).

Left the comment here as well: <https://github.com/home-assistant/architecture/discussions/692#discussioncomment-2494857>

Reply:
I thought i had resolved this and just unresolved to make sure we circle back and are happy with the naming. (I realize now maybe i didn't resolve it, but i can't remember)

Reply:
Yeah, I think it is fine and fitting. Anything else than "developer" would be good 👍